### PR TITLE
Feedback sprint6

### PR DESCRIPTION
**Pull request by MichielvdBerg**
"As a developer
I want to improve the 
feedback on my code
so that it is of better quality"

### CODE REVIEW

File: TU-Agent/agentKnowledge.pl [hunk 1]
```diff
@@ -8,19 +8,19 @@
...
 
 %The goals and how to achieve them.
 %we have to retrieve this only once and the goal will be dropped by hand
-getindicatorgoals :- false.
+getIndicatorGoals :- false.
 %an indicatorgoal is met if the current score is the target score
-indicatorgoal(Name, Target) :- indicator(Id, Name, Current, _), Target > 0, Current >= Target.
-indicatorgoal(Name, Target) :- indicator(Id, Name, Current, _), Target =< 0, Current =< Target.
+indicatorgoal(Name, Target) :- indicator(_, Name, Current, _), Target > 0, Current >= Target.
+indicatorgoal(Name, Target) :- indicator(_, Name, Current, _), Target =< 0, Current =< Target.
```
Comment (jzoon) [inline TU-Agent/agentKnowledge.pl]: Change to indicatorGoal?

File: TU-Agent/tygronEvents.mod2g [hunk 2]
```diff
@@ -2,40 +2,51 @@ use agentKnowledge as knowledge.
...
 	%update indicator when it does exist but is changed.
-	forall percept(indicators(X)), bel(indicatorlink(Weights), member(indicatorWeights(Id, Name, Score), Weights), indicators(Indicators), member(indicator(Id, Current, Target, _), Indicators)), bel(indicator(Id, Name, OldCurrent, Target))
-		do delete(indicator(Id, Name, OldCurrent, Target)) + insert(indicator(Id, Name, Current, Target)).
+	forall percept(indicators(X)), bel(indicatorlink(Weights), member(indicatorWeights(IndicatorID, Name, _), Weights), indicators(Indicators), member(indicator(IndicatorID, Current, Target, _), Indicators)), bel(indicator(IndicatorID, Name, OldCurrent, Target))
+		do delete(indicator(IndicatorID, Name, OldCurrent, Target)) + insert(indicator(IndicatorID, Name, Current, Target)).
 	
 	%set up the goals for the indicators
-	forall bel(indicator(Id, Name, Current, Target)), goal(getindicatorgoals)
+	forall bel(indicator(_, Name, _, Target)), goal(getIndicatorGoals)
 		do adopt(indicatorgoal(Name, Target)).
```
Comment (tvleest) [inline TU-Agent/tygronEvents.mod2g]: indicatorGoal

File: TU-Agent/tygronEvents.mod2g [hunk 3]
```diff
@@ -2,40 +2,51 @@ use agentKnowledge as knowledge.
...
+		do insert(indicator(IndicatorID, Name, Current, Target)).
 	%update indicator when it does exist but is changed.
-	forall percept(indicators(X)), bel(indicatorlink(Weights), member(indicatorWeights(Id, Name, Score), Weights), indicators(Indicators), member(indicator(Id, Current, Target, _), Indicators)), bel(indicator(Id, Name, OldCurrent, Target))
-		do delete(indicator(Id, Name, OldCurrent, Target)) + insert(indicator(Id, Name, Current, Target)).
+	forall percept(indicators(X)), bel(indicatorlink(Weights), member(indicatorWeights(IndicatorID, Name, _), Weights), indicators(Indicators), member(indicator(IndicatorID, Current, Target, _), Indicators)), bel(indicator(IndicatorID, Name, OldCurrent, Target))
+		do delete(indicator(IndicatorID, Name, OldCurrent, Target)) + insert(indicator(IndicatorID, Name, Current, Target)).
 	
 	%set up the goals for the indicators
-	forall bel(indicator(Id, Name, Current, Target)), goal(getindicatorgoals)
+	forall bel(indicator(_, Name, _, Target)), goal(getIndicatorGoals)
```
Comment (jzoon) [inline TU-Agent/tygronEvents.mod2g:42]: Again indicatorGoal, not indicatorgoal

**Comment by jzoon** [comment]
Approved after the changes

**Pull request merged**